docs/notebooks/0_introduction.ipynb

```diff
@@ -610,7 +505,7 @@
     "\n",
     "https://pz-server-dev.linea.org.br/product/6_simple_training_set\n",
     "\n",
-    "<font color=red> WARNING:</font> The URL works only with the internal name, **not** with the **id** number. \n"
+    "<font color=red> WARNING:</font> The URL works only with the **complete internal name**, not with just the **id** number. \n"
    ]
   },
   {
```

Comment:
"The tabular data is alocated in the attribute `data`". I think it should be "allocated".